builtin/settingtypes.txt

```diff
@@ -2314,6 +2314,9 @@ show_advanced (Show advanced settings) bool false
 #    Changing this setting requires a restart.
 enable_sound (Sound) bool true
 
+#    Fake setting used to handle DPI changes at runtime.
+dpi_change_notifier (DPI change notifier) int 0
```

Comment:
if it's not real I wouldn't document it here

Reply:
OK

It is saved to minetest.conf though. I could add code to avoid that, but that's probably not worth it.